acceptance/acceptance_test.go

```diff
@@ -211,7 +211,21 @@ func testAccept(t *testing.T, InprocessMode bool, singleTest string) int {
 				t.Parallel()
 			}
 
-			runTest(t, dir, coverDir, repls.Clone(), config, configPath)
+			expanded := internal.ExpandEnvMatrix(config.EnvMatrix)
+
+			if len(expanded) == 1 && len(expanded[0]) == 0 {
+				runTest(t, dir, coverDir, repls.Clone(), config, configPath, expanded[0])
+			} else {
+				for _, envset := range expanded {
+					envname := strings.Join(envset, "/")
```

Comment:
```suggestion
					envname := strings.Join(envset, "_")
```

Use of `/` normally indicates a path.

Reply:
"_" is ambiguous, env vars frequently contain "_". (In principle they can contain "/", but less frequently for this use case).

Reply:
Maybe `&` then?

Reply:
It's also the separator for nested calls to `t.Run`, so it may be ambiguous. `+` is another option.

Reply:
> Maybe & then?

This has special meaning in bash, might require extra quoting. But also see comment below - forward slash use is intentional.

> It's also the separator for nested calls to t.Run, so it may be ambiguous.

This _does_ generate nested test cases, with multiple levels.

e.g. if you have mytest/A=B/C=D then 'mytest/A=B' is group for all test cases with A=B.

This will work well with -run which treats `/` specially. You can do -run '/DEFAULT_SPARK_VERSION=.*13.*/` and this will work because regex is applied to one subtest, not to the whole path.